pallets/rc-migrator/src/staking/nom_pools.md

```diff
@@ -0,0 +1,33 @@
+# Pallet Nomination Pools
+
+The nomination pools pallet has 15 storage items of which 14 can be migrated without any
+translation.
+
+# Storage Values
+
+All nine storage values are migrated as is in a single message.
+
+# Storage Maps
+
+The `BondedPools` map needs translation for its commission logic. The Commission struct contains an
+absolute relay timestamp
```

Comment:
Is it not possible to rework this to continue using the `RcBlockNumberProvider` as a clock? Is it just because this hasn't been completed yet or is there a reason not to?

Reply:
I dont see an open MR to do it, so I assume that it will be done in the future. Generally yes, it can very well use the relay block number (and should). We just work off the state of the SDK that we have for now.

Reply:
Yea I'll get somebody on it. For these ones which haven't been completed yet but which are intended, I think you can write the migration and keep the number as is, but make a note and we can come back to it if it looks like the pallet won't be ready in time. Might save you a bit of time for these pallets where the mapping introduces some extra work